src/themes/website/styles/widgets/widgets.scss

```diff
@@ -18,3 +18,29 @@
 @import "fui/operation-details.scss";
 @import "fui/apis-products-lists.scss";
 @import "fui/reports.scss";
+
+/**
+ * If you need to make sure that custom defined styles do not interfere with widgets,
+ * this is a good place to do it. Make sure to add !important to insure overriding.
+ */
+[data-bind="widget: widget"] {
+  .button {
+    margin: 10px 20px 10px 0 !important;
+    padding: 9px 50px !important;
```

Comment:
50px seems like a very big number. Are you sure?

Reply:
Just checked again, it is indeed 50px
![image](https://github.com/user-attachments/assets/960214ef-e42e-4d97-af7b-72301a81b300)